cmd/system-probe/modules/network_tracer.go

```diff
@@ -68,7 +70,7 @@ var NetworkTracer = module.Factory{
 			startTelemetryReporter(cfg, done)
 		}
 
-		return &networkTracer{tracer: t, done: done}, err
+		return &networkTracer{tracer: t, done: done, maxConnsPerMessage: cfg.MaxConnsPerMessage}, err
```

Comment:
I know this isn't new, but I think this config value really belongs in `pkg/network/config` since it is network-specific.

Reply:
I'm guessing it was left at the top-level for ease of process-agent, so I guess leave it for now.